components/_previews/_preview--nogrid.njk

```diff
@@ -15,6 +15,7 @@
   {% if _config.project.environment.production %}
   <link rel="stylesheet" href="https://dev.assets.emblstatic.net/vf/develop/css/styles.css">
   {% endif %}
+  <style>body { margin: 0; }</style>
```

Comment:
Is this necessary, or was is it just a local development artefact?

Reply:
yeah - the body element in fractal has margin - os it makes things look like they have more space than they would in a real web page - removing it - makes things like `vf-body` work 'as expected'.

Reply:
Gotcha. Fractal thingy 😄